src/btree/bt_vrfy.c

```diff
@@ -170,30 +170,24 @@ __wt_verify(WT_SESSION_IMPL *session, const char *cfg[])
     const char *name;
     bool bm_start, quit;
 
-#if 0
-    /* FIXME-WT-6263: Temporarily disable history store verification. */
     uint32_t session_flags;
```

Comment:
Can we move these back into the declaration block above?
The only reason I moved them down here is so I could `#ifdef` them out as a group.

Reply:
fixed